package.json

```diff
@@ -24,14 +24,18 @@
     "dev": "lerna run --parallel --scope @firebase/* --scope firebase dev",
     "build": "lerna run --scope @firebase/* --scope firebase build",
     "build:changed": "ts-node-script scripts/ci-test/build_changed.ts",
+    "build:ci:auth": "lerna run build:deps --scope '@firebase/auth*'",
```

Comment:
Same comment regarding putting the command directly in the workflow.

Reply:
Done.